scripts/init-simapp.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+SIMD_BIN=${SIMD_BIN:=$(which simd 2>/dev/null)}
+
+if [ -z "$SIMD_BIN" ]; then echo "SIMD_BIN is not set. Make sure to run make install before"; exit 1; fi
+echo "using $SIMD_BIN"
+if [ -d "$($SIMD_BIN config home)" ]; then rm -r $($SIMD_BIN config home); fi
+$SIMD_BIN config set client chain-id demo
```

Comment:
maybe we should use a chain id that has a suffix incase things like ibc upgrades want to be tested.

```suggestion
$SIMD_BIN config set client chain-id simapp-1
```